egs/wsj/s5/steps/dict/prune_pron_candidates.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python
```

Comment:
For the scripts, like this one, where you think it's unlikely that a user will want to call them directly, please put them in a subdirectory internal/.  This makes it easier to look through the list of top-level scripts.